modules/random-list/random-list.nu

```diff
@@ -0,0 +1,115 @@
+# Generate a random boolean list.
+export def "random-list bool" [
+    list_length: int, # A length of the list
+    bias: float = 0.5 # A probability of "true"
+] {
+    if $bias < 0 or $bias > 1 {
+        error make {
+            msg: "invalid probability: must be between 0 and 1"
+        }
+    }
+    if $list_length < 0  {
+        error make {
+            msg: "invalid list length: must be greater than 0"
+        }
+    }
+
+    1..$list_length | each {|it|
+        random bool --bias $bias
+    }
+}
+
+# Generate a random char list.
+export def "random-list chars" [
+    list_length: int, # A length of the list
+    length: int = 5 # A length of the string
```

Comment:
this might be clearer as `--lenght`?

Reply:
I've replaced 2nd, 3rd and so on parameters with options in all `def`s. 😸

Reply:
amazing :open_mouth: